Sources/StreamVideo/WebRTC/StatsReporter.swift

```diff
@@ -0,0 +1,168 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import WebRTC
+
+enum StatsReporter {
```

Comment:
Why not a have all those calculations as part of the CallStatsReport's init `.init(reports: [RTCStatisticsReport?], datacenter: String)` and all the calculations happening in the init? (reduce the static load)

Reply:
`RTCStatisticsReport` can't be instantiated, which makes it impossible to test. I have it split so I can test some chunks easily (the parts where we work with dictionaries).